bindings/pydeck/docs/scripts/embed_examples.py

```diff
@@ -8,13 +8,7 @@
 import subprocess
 import sys
 
-from const import (
-    DECKGL_URL_BASE,
-    EXAMPLE_GLOB,
-    GALLERY_DIR,
-    HTML_DIR,
-    HOSTED_STATIC_PATH,
-)
+from const import DECKGL_URL_BASE, EXAMPLE_GLOB, GALLERY_DIR, HTML_DIR, HOSTED_STATIC_PATH
```

Comment:
Seems like there are some lint changes? Did the config change, e.g. max line length? If so might be cleaner as a separate PR.

Reply:
Yes, appears to have been done by mistake and seems like I mixed it in with another commit.